packages/data-controls/src/index.ts

```diff
@@ -35,45 +37,67 @@ export function apiFetch( request ) {
  * Control for resolving a selector in a registered data store.
  * Alias for the `resolveSelect` built-in control in the `@wordpress/data` package.
  *
- * @param {Array} args Arguments passed without change to the `@wordpress/data` control.
+ * @param storeNameOrDescriptor
+ * @param selectorName
```

Comment:
Same for `selectorName`. I realize it feels redundant to have "Name of the selector", but some developers find it easier to directly look at the description first when reading code.

Feel free to borrow from the `resolveSelect()` JSdoc block.